vizro-ai/docs/pages/user-guides/create-complex-dashboard.md

```diff
@@ -1,5 +1,9 @@
 # Generate a complex dashboard
 
+!!! warning "Alpha Feature"
+    This feature is currently in **Alpha**. It is stable but not officially released yet. We welcome feedback, but please be aware that it may still undergo minor adjustments.
```

Comment:
We need to say how to leave feedback (e.g. issue on GitHub) and link to that way.